src/components/create-appointment-fourth/CreateAppointmentFourth.tsx

```diff
@@ -0,0 +1,144 @@
+  }, [caregiverFilter, getFilteredCaregivers, translate]);
+
+  return (
+    <Background>
+      <FilterContainer>
+        <BaseText>{translate('createAppointmentFourth.location')}</BaseText>
+        <CreateAppointmentFourthAutocomplete onLocationChange={handleLocationChange} />
+        <BaseText>{translate('createAppointmentFourth.availability')}</BaseText>
+        <StyledFormControlLabel
+          control={
+            <Switch
+              value={caregiverFilter.isShowAvailableCaregivers}
+              checked={caregiverFilter.isShowAvailableCaregivers}
+              onChange={handleSwitchChange}
+              name="isShowAvailableCaregivers"
+            />
+          }
+          label={translate('createAppointmentFourth.showAvailableCaregivers')}
+        />
+        <StyledFormControlLabel
+          control={
+            <Switch
+              value={caregiverFilter.isOpenToSeekerHomeLiving}
+              checked={caregiverFilter.isOpenToSeekerHomeLiving}
+              onChange={handleSwitchChange}
+              name="isOpenToSeekerHomeLiving"
+            />
+          }
+          label={translate('createAppointmentFourth.openToLivingInClientsHouses')}
+        />
+        <BaseText>{translate('createAppointmentFourth.servicesTitle')}</BaseText>
+        <FormGroup>
+          {caregiverFilter.services.map((service) => (
+            <StyledFormControlLabel
+              key={service.label}
+              control={
+                <Checkbox
+                  checked={service.checked}
+                  onChange={handleServicesChange}
+                  name="services"
+                  value={service.label}
+                />
+              }
+              label={translate(`createAppointmentFourth.services.${service.label}`)}
+            />
+          ))}
+        </FormGroup>
+      </FilterContainer>
+      <CaregiverListContainer>
+        <List>
+          {!!filteredCaregivers &&
+            filteredCaregivers.map((caregiver) => (
+              <ListItem
+                key={caregiver.id}
+                secondaryAction={
+                  <IconButton
+                    edge="end"
+                    aria-label="open-drawer"
+                    onClick={(): void => handleDrawerOpen(caregiver.id)}
+                  >
+                    <RightAction />
+                  </IconButton>
+                }
+              >
+                <ListItemAvatar>
+                  <StyledAvatar
+                    src={getMockCaregiverAvatar(SMALL_CAREGIVER_AVATAR_SIZE)}
+                    alt={`${caregiver.firstName} ${caregiver.lastName}`}
+                  />
+                </ListItemAvatar>
+                <StyledListItemText
+                  primary={`${caregiver.firstName} ${caregiver.lastName}`}
+                  secondary={
+                    <>
+                      <MonetizationOn />{' '}
```

Comment:
could you please remove {' '} ?

Reply:
I can't remove that because eslint and prettier limits max length of code in a line